kmod/src/scoutfs_trace.h

```diff
@@ -426,11 +425,15 @@ TRACE_EVENT(scoutfs_trans_write_func,
 
 TRACE_EVENT(scoutfs_release_trans,
 	TP_PROTO(struct super_block *sb, void *rsv, unsigned int rsv_holders,
-		 struct scoutfs_item_count *res,
-		 struct scoutfs_item_count *act, unsigned int tri_holders,
+		 int reserved_items,
```

Comment:
Oh, if we're removing the item accounting, we're not just removing the arguments to hold and release.  We'd be removing all the accounting in the transactions too.  That's the book keeping to remove :).  So no more reserved/actual fields.